README.md

```diff
@@ -75,76 +75,65 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.31 - January 2021 Update
+### 0.33 - February 2021 Update
```

Comment:
It's march

Reply:
@Aaron-Junker 
it's still the Feb release even if we ship it a few days later, otherwise how are we going to call 0.35 when released at the end of March?

Reply:
this is what we'd still refer to as a feb update